pkg/k8shandler/rsyslog_test.go

```diff
@@ -79,7 +79,7 @@ func TestRsyslogPodSpecHasTaintTolerations(t *testing.T) {
 			},
 		},
 	}
-	podSpec := newFluentdPodSpec(cluster, "test-app-name", "test-infra-name")
+	podSpec := newFluentdPodSpec(cluster, "test-app-name", "test-infra-name", nil)
```

Comment:
i realize this is outside the scope of this PR but why is there the creation of a fluentd pod spec in the rsyslog test?

Reply:
I thought the test was written to cover some case, but I can't see a valid reason...  @richm, is it ok to replace newFluentdPodSpec with newRsyslogPodSpec?

Reply:
yes - looks like a copypasta error

Reply:
i'm surprised this didn't still fail... `newFluentdPodSpec` had its arguments updated and this should have been fixed by calling `newRsyslogPodSpec` instead of just reverting adding `,nil`